main.css

```diff
@@ -0,0 +1,485 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  /* font-family: "Roboto Condensed", sans-serif; */
```

Comment:
- [ ] Remove commented-out code. You can keep it in another branch if you still want to go back to it.